helix-term/src/commands/typed.rs

```diff
@@ -2086,6 +2086,11 @@ fn sort_impl(
 
     let selection = doc.selection(view.id);
 
+    if selection.len() == 1 {
+        cx.editor.set_warning("Sorting requires multiple selections. Hint: split selection first");
+        return Ok(());
```

Comment:
We can return this in one line as an error

```suggestion
        bail!("Sorting requires multiple selections. Hint: split selection first");
```

Since we're bailing without doing anything I think an error seems appropriate here